src/universal/routes/logout.js

```diff
@@ -1,7 +1,16 @@
-export default {
+import {makeWhitelistedReducer} from 'universal/redux/makeReducer';
+
+const reducerWhitelist = [
+  'notifications'
+];
+
+export default (store) => ({
   path: '/logout',
   getComponent: async(location, cb) => {
     const component = await System.import('universal/containers/Logout/LogoutContainer');
+    const newReducer = makeWhitelistedReducer(reducerWhitelist);
+    store.replaceReducer(newReducer);
```

Comment:
I think [this](https://github.com/ParabolInc/action/pull/157/files#diff-3f48aadcc1b3be42a6f79e920d133c59R18)'ll do it.